book/blocks.md

```diff
@@ -1038,19 +1029,18 @@ From [Nedcamp video on caching by Kelly Lucas, November 2018](https://www.youtub
 
 In a twig template, let's you say you just want to render one field (but not the others), Drupal may not be aware if the content has changed, and will sometimes show old cached content. You can define a view mode or tweak the twig template a smidge with something like this:
 
-
-  \{\% set blah = content\|render \%\}
-
+```twig
```

Comment:
I hope that jekyll doesn't barf when it sees this.  I've had problems with it getting so confused with twig code that I've sometimes had to resort to images of that code.

Reply:
When I added it my editor didn't produce any errors, but forgot about jekyll!
I did notice there are already some twig code blocks in the codebase, so I thought it would be ok?
We need to get an official list from somewhere.

Reply:
I'll find out soon enough.  As soon as I merge these changes, Github actions will let me know which lines it doesn't like.    I haven't found a comprehensive list of what to use.  Probably using `javascript` instead of `js` is correct.  Thanks for all the hard work.

Reply:
Cool, let's give it a go, if it fails we'll have some tags to change :)